src/app/users/views/usersList.ejs

```diff
@@ -123,7 +123,7 @@
                                     </fieldset>
                                     <div class="govuk-button-group">
                                         <button type="submit" class="govuk-button" name="applyFilter" value="applyFilter" >Apply filter</button>
-                                        <button type="submit" class="govuk-button govuk-button--secondary govuk-button--secondary-dark <%=locals.selectedOrganisations.length > 0 ? '' : 'govuk-button--disabled' %>" name="removeFilter" value="removeFilter" <%=locals.selectedOrganisations.length > 0 ? '' : 'disabled' %> > Remove filter</button>
+                                        <button type="submit" class="govuk-button govuk-button--secondary govuk-button--secondary-dark" name="removeFilter" value="removeFilter" <%=locals.selectedOrganisations.length > 0 ? '' : 'disabled aria-disabled=true' %> > Remove filter</button>
```

Comment:
`true` should be in double quotes; eg. `aria-disabled="true"`

Reply:
This is now fixed